test/integration/component/test_ldap_autoImport.py

```diff
@@ -0,0 +1,663 @@
+
+        else:
+            self.delflag5 = 1
+            self.domainRes3 = self.apiClient.createDomain(self.domain3)
+        self.assertEquals(self.delflag5, 1, "Domain creation failed,please check the configuration")
+        #return
+
+        #here link ldap to domain
+
+        self.ldaplink3 = linkDomainToLdap.linkDomainToLdapCmd()
+        self.ldaplink3.domainid = self.domainRes3.id
+        self.ldaplink3.accounttype = self.services["configurableData"]["link_ldap_details"]["accounttype"]
+        self.ldaplink3.name = self.services["configurableData"]["link_ldap_details"]["name"]
+        self.ldaplink3.type = self.services["configurableData"]["link_ldap_details"]["type"]
+
+        if self.services["configurableData"]["link_ldap_details"]["admin"] is not None:
+            self.ldaplink3.admin = self.services["configurableData"]["link_ldap_details"]["admin"]
+
+        if self.ldaplink3.domainid == "" or self.ldaplink3.accounttype == "" or self.ldaplink3.name == "" \
+                or self.ldaplink3.type == "":
+            self.debug("Please rerun the test by providing values in link_ldap configuration user details")
+            self.skipTest("Please rerun the test by providing proper values in configuration file(link ldap)")
+
+        else:
+            self.delflag6 = 1
+            self.ldaplinkRes3 = self.apiClient.linkDomainToLdap(self.ldaplink3)
+        self.assertEquals(self.delflag6, 1, "Linking LDAP failed,please check the configuration")
+
+        loginRes = self._checklogin(
+            self.services["configurableData"]["link_ldap_details"]["linkLdapUsername"],
+            self.services["configurableData"]["link_ldap_details"]["linkLdapPassword"],
+            self.domain2.name,
+            method="POST")
+        self.debug(loginRes)
+        self.assertEquals(loginRes, 1, self.reason)
+        #domain 3 ends here
+
+        #deleting domain 2 and 3 here
+        try:
+                deleteDomain2 = deleteDomain.deleteDomainCmd()
+                deleteDomain2.id = self.domainRes2.id
+                deleteDomain2.cleanup = 'true'
+
+                domain_name = self.domainRes2.name
+
+                self.apiClient.deleteDomain(deleteDomain2)
+               # self.domainRes.delete(self.apiclient, cleanup=True)
+                self.debug(
+                    "Deleted the the following account name %s:" %
+                    domain_name)
+        except Exception as e:
+                raise Exception("Warning: Exception during cleanup : %s" % e)
+
+
+
+        try:
+                deleteDomain3 = deleteDomain.deleteDomainCmd()
+                deleteDomain3.id = self.domainRes3.id
+                deleteDomain3.cleanup = 'true'
+
+                domain_name = self.domainRes3.name
+
+                self.apiClient.deleteDomain(deleteDomain3)
+               # self.domainRes.delete(self.apiclient, cleanup=True)
```

Comment:
Please remove code, do not comment it. It can be retrieved if necessary from Git.